pom.xml

```diff
@@ -2161,9 +2182,6 @@
           <shadedArtifactAttached>false</shadedArtifactAttached>
           <artifactSet>
             <includes>
-              <!-- At a minimum we must include this to force effective pom generation -->
-              <include>org.spark-project.spark:unused</include>
-
               <include>org.eclipse.jetty:jetty-io</include>
```

Comment:
Hmm, I don't seem to be able to find those classes after unzipping the JAR in the /jars folder.